01_fib/alanhillgemann/main_test.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+	main()
+	expected := strconv.Quote("0\n1\n1\n2\n3\n5\n8\n13\n")
+	actual := strconv.Quote(buf.String())
+	assert.Equalf(t, expected, actual, "expected %v, actual %v was not sorted", expected, actual)
+}
+
+var testCases = map[string]struct {
+	input    int
+	expected string
+}{
+	"Neg": {input: -5, expected: "0\n"},
+	"0":   {input: 0, expected: "0\n"},
+	"1":   {input: 1, expected: "0\n1\n"},
+	"2":   {input: 2, expected: "0\n1\n1\n"},
+	"7":   {input: 7, expected: "0\n1\n1\n2\n3\n5\n8\n13\n"},
```

Comment:
Requested output in lab assignment was `1,  1, ...` not `0, 1, 1` - which as arguably a bad choice in the first place, however, please add at least a comment why you chose to deviate from the assignment or fix.

Reply:
Fixed